package.json

```diff
@@ -22,12 +22,13 @@
     "bin/*.js",
     "lib/**/*.js"
   ],
-  "standard": {
+  "xo": {
     "ignore": [
-      "/tap-snapshots/",
-      "**/fixtures/**",
-      "**/test/build/*"
-    ]
+      "self-coverage",
```

Comment:
You don't need to add this because it's in .gitignore and xo respects it for the excludes

Reply:
Basically this should be enough, but please confirm:

```
    "ignores": [
      "**/fixtures/**"
    ],
```

Reply:
Actually `tap-snapshots` isn't needed either (the latest release of xo ignores it by default).